FBSnapshotsViewer/Extensions/FileManager+Move.swift

```diff
@@ -10,7 +10,13 @@ import Foundation
 
 extension FileManager {
     func moveItem(at fromURL: URL, to toURL: URL) throws {
-        try self.removeItem(at: toURL)
+        try deleteItem(at: toURL)
```

Comment:
I agree, deleteIfExists would be much better and I'd been thinking exactly the same thing since making the pull request. The reason for it being a separate method is because it's used both in moveItem here, and is called directly from the removeTestImages function in SnapshotTestResultSwapper.swift.

Reply:
I guess you mean the overhead of checking if file exists? I think it's fine and it should not add any performance cost 😄